presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4264,4 +4264,12 @@ public void testValuesWithTimestamp()
         MaterializedRow row = rows.get(0);
         assertEquals(row.getField(0), row.getField(1));
     }
+
+    @Test
+    public void testFilterPushdownWithAggregation()
+            throws Exception
+    {
+        assertQuery("SELECT * FROM (SELECT count(*) FROM orders) WHERE 0=1");
```

Comment:
Nice, I did not know that. So, as I understand H2 is loaded with data matching to tpch.tiny, right?